campaignSpec.md

```diff
@@ -100,3 +116,8 @@ Rules that apply to submitting events
 * `modified`: number, UTC timestamp in milliseconds, changed every time modifiable property is changed
 
 [ipfs]: https://ipfs.io/
+
+
+##### Dynamic Price Adjustment
```

Comment:
add a warning that this is not in place yet 

`*WARNING:* this is supported yet`

Since we're not planning to add it immediately